From: Ben Hutchings <ben@decadent.org.uk>
Date: Sun, 26 Nov 2017 21:44:30 +0000
Subject: libsas: Avoid ABI change in 3.2.95
Forwarded: not-needed

Commit a3a142524aa4 "libsas: prevent double completion of scmds from eh"
added a new enumerator to enum exec_status.  This isn't a real ABI change
because its value was already being stored in enum exec_status.  Hide it
from genksyms.

---
--- a/include/scsi/libsas.h
+++ b/include/scsi/libsas.h
@@ -452,7 +452,9 @@ enum exec_status {
 	/* The SAM_STAT_.. codes fit in the lower 6 bits, alias some of
 	 * them here to silence 'case value not in enumerated type' warnings
 	 */
+#ifndef __GENKSYMS__
 	__SAM_STAT_CHECK_CONDITION = SAM_STAT_CHECK_CONDITION,
+#endif
 
 	SAS_DEV_NO_RESPONSE = 0x80,
 	SAS_DATA_UNDERRUN,
